docs/process/releasing-updates.md

```diff
@@ -118,20 +118,19 @@ Here's an example of the previous changelog draft after it has been edited:
 }
 ```
 
-Once you've pushed up the version update and the changelog changes, you're ready to release! Get the others on the team to :thumbsup: in a PR if you're not sure. Note that any version change that does not have an associated changelog entry will not successfully release.
+Create a new branch off `master` to represent the work that will be released to users. Add your new changelog entries and version update, and push this branch to GitHub. This becomes the release branch, and lets other maintainers continue to merge into `master` without affecting your release.
```

Comment:
should we also add here that for `production` releases, it should be branched off of the last `beta` release branch?